.github/workflows/os-smoke-test.yml

```diff
@@ -0,0 +1,34 @@
+name: OS Support Smoke Test
+
+on:
+  push:
+    branches: [ develop ]
+  pull_request:
+    branches: [ develop ]
+
+jobs:
+  build:
+    name: Build Zeebe and run smoke test on ${{ matrix.os }} to ensure development support.
+    timeout-minutes: 10
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [macos-latest, windows-latest, ubuntu-latest]
```

Comment:
💭 I think it's best to test for the latest versions of each os, but this means builds are not fully reproducible. Not sure if that's really a problem though